src/assets/scripts/math/circle.js

```diff
@@ -64,3 +64,27 @@ export const fix_angle = (radians) => {
 
     return radians;
 };
+
+/**
+ * normalize angles to within 0° - 360°
+ * @param  {number} degrees an angle
+ * @return {number}         an angle within [0,360]
+ */
+export const degrees_normalize = (degrees) => {
+    if (degrees < 0) {
+        return 360 + (degrees % 360);
+    }
+    else return degrees % 360;
+};
+
+/**
+ * normalize angles to within 0 - 2π
```

Comment:
just to clarify, both of those doc gens actually _generate_ html pages based on the doc blocks within the code.